bin/run.sh

```diff
@@ -4,4 +4,4 @@ TOP=$(dirname $( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd
 
 cd "$TOP"
 
-exec ./bin/run.py "${@}"
```

Comment:
Shouldn't the `run.py` file be removed?

Reply:
I removed the script and the python package installation from the Dockerfile.